GDCpp/GDCpp/Extensions/Builtin/StringTools.h

```diff
@@ -26,12 +26,16 @@ gd::String GD_API StrRepeat(const gd::String& str, std::size_t repCount);
 std::size_t GD_API StrLen(const gd::String& str);
 int GD_API StrFind(const gd::String& str, const gd::String& findwhat);
 int GD_API StrRFind(const gd::String& str, const gd::String& findwhat);
+int GD_API StrFindLast(const gd::String& str, const gd::String& findwhat);

```

Comment:
This is just adding the declaration of a function ;) Instead, let's rename StrRFind to StrFindLast (and some for the other functions), both here and in StringTools.cpp